hardware/run_all_sims.py

```diff
@@ -0,0 +1,74 @@
+from subprocess import run
```

Comment:
Drop the .py extension from the file and make this a script (add "#! /usr/bin/env python3" as the first line).

Reply:
`removesuffix` requires python 3.9, so I used `/home/ff/eecs151/miniconda3/bin/python3` instead.